string lang
string options
string text
---
int32 result
